src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,172 @@
 package core.basesyntax;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int DOUBLE_SIZE = 2;
+    private Node<K,V> [] table;
+    private int size;
```

Comment:
Using public access modifiers everywhere: The field 'size' should be private to adhere to encapsulation principles.